tests/test_utils.py

```diff
@@ -692,14 +713,19 @@ def test_nullcontext() -> None:
         ),
         (
             lambda is_optional, missing: FloatNode(
-                value=10 if not missing else "???", is_optional=is_optional
+                value=10.0 if not missing else "???", is_optional=is_optional
```

Comment:
If this is a non related change, it would be better in a separate commit

Reply:
Sorry about that. It was an unrelated change. It's been reverted now.

Reply:
I've followed up in PR https://github.com/omry/omegaconf/pull/853.

Reply:
I've followed up in PR https://github.com/omry/omegaconf/pull/853.